StreetHawk/Classes/Crash/Private/SHCrashBridge.m

```diff
@@ -117,14 +115,4 @@ + (void)installUpdateSucceededForCrash:(NSNotification *)aNotification
     }
 }
 
-+ (NSString*)getMD5Checksum:(NSString*)content
```

Comment:
this name is better than md5

getMD5Checksum is a better function name than md5